server/routes/index.js

```diff
@@ -271,8 +271,37 @@ module.exports = function Index ({ authenticationMiddleware }) {
       req.session.formInvalid = true
       redirectUrl = `/match/defendant/${req.params.caseNo}/manual`
     } else {
-      // @TODO: Redirect to new route to check record data - see PIC-618
-      redirectUrl = `/match/defendant/${req.params.caseNo}/manual`
+      redirectUrl = `/match/defendant/${req.params.caseNo}/confirm/${req.body.crn}`
+    }
+    res.redirect(redirectUrl)
+  })
+
+  router.get('/match/defendant/:caseNo/confirm/:crn', health, defaults, async (req, res) => {
+    const templateValues = await getCaseAndTemplateValues(req)
+    const detailResponse = await getDetails(req.params.crn)
+    templateValues.title = 'Link an nDelius record to the defendant'
+    templateValues.details = {
+      ...detailResponse
+    }
+    templateValues.session = {
+      ...req.session
+    }
+    res.render('match-manual', templateValues)
+  })
```

Comment:
There's no mention on the ticket but we probably need to have an error displayed to the user if the crn can't be found, otherwise the user will just see an empty table.

Reply:
Yes, you're right. I thought of this but forgot to mention it. I'll raise it with Charan